src/website.rs

```diff
@@ -222,6 +231,77 @@ impl Benchmark {
             "cache_misses",
         ]
     }
+
+    fn calculate_means(timings: &[TimingRecord]) -> SimpleRuntimeMetrics {
+        let calc = |f: fn(&TimingRecord) -> f64| {
+            calculate_sample_mean(&timings.iter().map(f).collect::<Vec<_>>())
+        };
+        SimpleRuntimeMetrics {
+            nanoseconds: calc(|t| t.metrics.nanoseconds.median.raw()),
```

Comment:
Can I suggest moving the `t.metrics` and `.median.raw()` bits here into `calc`'s definition? Maybe something like this:

```
let calc = |f: fn(&RuntimeMetrics) -> f64| {
	let times = timings.iter().map(|t| f(&t.metrics).median.raw()).collect::<Vec<_>>();
	calculate_sample_mean(&times)
};
```

should let you write each field as `calc(|m| m.instructions)` etc., as in the other initializers.